lib/contracts/erc20fixed.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.6.8;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ERC20Fixed is ERC20 {
+    constructor() public ERC20("Token", "TKN") {
+        _mint(msg.sender, 1000000);
+    }
+}
```

Comment:
Can these contracts be precompiled in the transaction generator container instead of compiling them on start?

Reply:
no, they need to be compiled at the start so that splunk can use the ABI to decode the transaction & event data.